app/Console/Commands/RevokeRepositoryAccessForExpiredLicensesCommand.php

```diff
@@ -20,19 +28,43 @@ public function handle(GitHubApi $gitHubApi)
             ->whereExpired()
             ->cursor()
             ->each(function (License $license) use ($gitHubApi) {
-                if ($license->purchase->user->licenses()->whereNotExpired()->where('purchasable_id', $license->purchasable_id)->exists()) {
+                if ($license->purchase->user->licenses()
+                    ->whereNotExpired()
+                    ->where('purchasable_id', $license->purchasable_id)
+                    ->exists()) {
                     // User has another license for this repo
```

Comment:
Let's remove the need for comments. Move the code above to a function called `userHasAnotherLicense` which returns a boolean.